content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+10 reasons you should give another chance to Firefox Nightly
+############################################################
+
+:slug: give-firefox-nightly-a-new-chance
+:date: 2016-12-22
+:lang: en
+:url: give-firefox-nightly-a-new-chance
+:summary:
+    Help us make the best Mozilla Firefox of all the times by mid 2017!
+:status: draft
+
+What is Nightly?
+================
+
+Firefox Nightly build also known as Nightly is a version of Firefox
+built from the source code everyday at night.
+
+This means that Firefox updates itself everyday!
+
+
+Why aren't you using Nightly today?
+===================================
+
+Anyone wants a stable Firefox and you don't have time for a crappy
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
+is getting lower really fast everyday.
+
+Also most of the bugs that you might have will be fixed the day after.
```

Comment:
I think you need more "connectors". How about "Besides," or "Additionally,"? (I'd replace "day after" with "the next day".)